ptsprojects/zephyr/iutctl.py

```diff
@@ -334,19 +334,20 @@ def init_stub():
     ZEPHYR = ZephyrCtlStub()
 
 
-def init(kernel_image, tty_file, board=None, use_rtt2pty=False):
+def init(kernel_image, tty_file, board=None, board_id=None, use_rtt2pty=False):
     """IUT init routine
 
     kernel_image -- Path to Zephyr kernel image
     tty_file -- Path to TTY file, if specified QEMU will not be used and
                 BTP communication with HW DUT will be done over this TTY.
     board -- HW DUT board to use for testing. This parameter is used only
              if tty_file is specified
+    board_id -- Serial number of the HW DUT board to use for testing.
```

Comment:
Should we use the same naming "IUT" as same as other places above? e.g. Serial number of IUT

Reply:
I saw here the comments of parameters are different. Like board is also mentioned as HW DUT here which is different from the other places so I tried to make board_id adapt to it